### ISSUE TITLE
WCAG AA and AAA - WCAG 2.0 & 2.1

### ISSUE THREAD

**Issue by mgifford** [dwp/accessibility-manual#28]
I am looking at https://accessibility-manual.dwp.gov.uk/best-practice/wcag-aa-and-aaa

And I'm not sure that's the right approach. I think at this point it is useful talking about WCAG 2.0, 2.1, 2.2 & 3.0. 

It isn't clear from this document that WCAG 2.0 requirements are included in WCAG 2.1 (but they are).

AAA should be aspirational, and AA the legal floor.  Every government site should be AA, and depending on the audience of the site it may be important to prioritize which AAA criteria a department strives to achieve. You kind of get to this in the document but to meet WCAG AA, all WCAG A requirements need to be fulfilled. 

This might be accurate:

> Some of the impacts of making your service AAA will involve changing how a user interacts with your service and will go against security and GDS patterns, however, some will be fulfilled by default.

But folks seeing this will just stop reading any further.  If there are specifics about AAA requirements that go against GDS & security best practices, be specific. There should be an issue queue somewhere to track current best practices to addressing this. 

I'm not sure this is the best way to express this either:
> We have a legal requirement to meet AA, however to meet AAA we have to meet every single one of the WCAG 2.1 criteria.

I don't think a government website exists that can claim to be AAA compliant. Certainly the W3C isn't.  Being WCAG 2.x AAA compliant is next to impossible, and considerably increases the costs of building and maintaining a site.  Not that those aren't worth striving for, but nobody should be talking about AAA compliance, as it doesn't really exist.

**Comment by abbott567** [comment]
Thanks @mgifford. So this page came about because we had an issue (that may be specific to DWP) of teams grabbing the bull by the horns and then deciding they were going for AAA compliance.

When you say AAA should be aspirational and AA the legal floor, I don't think thats the right way to phrase it either. We shouldn't aspire for AAA across the board as W3C don't even recommend it. It's perhaps more like: `AA is the legal floor, but we should include elements of AAA`.

I also don't think a Gov website exists that can claim to be AAA compliant, an it shouldn't. The point of this page was to try and explain to those overly enthusiastic teams that AAA compliance is not actually better, and perhaps an impossible task. 

We do mention the criteria which can affect GDS patterns further down, so perhaps it just needs tidying up to be consistent. I think you've touched on a good point though about the 2.0 vs 2.1 thing. I would be more inclined to add it as an additional section, rather than change this one as it does serve a purpose (even if it's just for our own overly enthusiastic teams).

```
Some elements of AAA would likely require you to deviate from the GDS design system styles, such as:
- 1.4.6 Contrast
- 2.5.5 Target Size
```

**Comment by mgifford** [comment]
This all makes sense. Perhaps the key comes down to understanding the user.  The UK government is really a world leader in user research, so perhaps there is a way to highlight this. 

If through user research we identify that a AAA SC would be preferable, then we should strive for that.  Alternatively, something about a commitment to review AAA SC to see if it would be achievable and better serve users.  Not sure, so many folks just ignore the AAA requirements as they aren't required.
